test/functional/cmdLineTests/jfr/playlist.xml

```diff
@@ -63,7 +63,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			-DMETADATA_BLOB_PATH=$(Q)$(TEST_RESROOT)$(D)metadata.blob$(Q) \
 			-DRESJAR=$(Q)$(TEST_RESROOT)$(D)jfr.jar$(Q) \
 			-jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)jfrevents.xml$(Q) \
-			-explainExcludes -xids all,$(PLATFORM),$(VARIATION) -nonZeroExitWhenError; \
+			-explainExcludes -xids all,$(PLATFORM),$(VARIATION) -plats all,$(PLATFORM),$(VARIATION) -nonZeroExitWhenError; \
 			${TEST_STATUS}
```

Comment:
I don't think `$(VARIATION)` should appear be include in the `-plats` option.
Please split lines and list options alphabetically for improved readability:
```
			-jar $(CMDLINETESTER_JAR) \
			-config $(Q)$(TEST_RESROOT)$(D)jfrevents.xml$(Q) \
			-explainExcludes \
			-nonZeroExitWhenError \
			-plats all,$(PLATFORM) \
			-xids all,$(PLATFORM),$(VARIATION); \
			${TEST_STATUS}
```
Please do likewise with the command on lines 35-36.